eva_submission/xlsx/xlsx_validation.py

```diff
@@ -115,6 +130,38 @@ def semantic_validation(self):
             self.warning(f'In some Samples, Taxonomy and scientific names are inconsistent. TaxId - {correct_taxid_sc_name.keys()}')
             self.correct_taxid_scientific_name_in_metadata(correct_taxid_sc_name, self.metadata_file, self.metadata['Sample'])
 
+    def check_biosamples_accessions(self):
+        """Check that BioSample accessions exist and are public"""
+        for row in self.metadata['Sample']:
+            if row.get('Sample Accession'):
+                sample_accession = row.get('Sample Accession').strip()
+                try:
+                    _ = self.communicator.follows_link('samples', join_url=sample_accession)
+                except ValueError:
+                    self.error_list.append(
+                        f'In Sample, row {row.get("row_num")} BioSamples accession {sample_accession} '
+                        f'does not exist or is private')
+
+    def check_project_accessions(self):
+        """Check that ENA project accessions exists and are public"""
+        for project_row in self.metadata['Project']:
+            for column_name in ['Parent Project(s)', 'Child Project(s)', 'Peer Project(s)']:
+                if project_row.get(column_name):
+                    for project_acc in project_row[column_name].split(','):
+                        if not check_project_format(project_acc):
+                            self.error_list.append(
+                                f'In Project, row {project_row.get("row_num")}, {column_name}: {project_acc} is not a valid project accession')
+                            continue
+                        if not check_existing_project_in_ena(str(project_acc)):
+                            self.error_list.append(
+                                f'In Project, row {project_row.get("row_num")}, {column_name}: {project_acc} does not exist or is private')
+                column_name = 'Project Alias'
```

Comment:
I think this is indented one level too deep...
![image](https://github.com/EBIvariation/eva-submission/assets/5599307/82f27a23-cb2d-469d-b6ad-f79381fc99dc)